gomock_generator/README.md

```diff
@@ -1,4 +1,4 @@
-# GoMock Generator
+# GoMock Generator v1.2.1
```

Comment:
Can you modify to have the same pattern as the others?

==> Package `gomock_generator` v1.2.1